MonoGame.Framework/SDL/SDLGameWindow.cs

```diff
@@ -0,0 +1,197 @@
+// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+
+namespace Microsoft.Xna.Framework
+{
+    class SDLGameWindow : GameWindow, IDisposable
+    {
+        public override bool AllowUserResizing
+        {
+            get
+            {
+                return !this.IsBorderless && _reziable;
+            }
+            set
+            {
+                throw new Exception("SDL does not support changing resizable parameter. Please use static property Game.Resizable before your game creation.");
+            }
+        }
+
+        public override Rectangle ClientBounds
+        {
+            get
+            {
+                int x = 0, y = 0, w, h;
+
+                SDL.SDL_GetWindowSize(Handle, out w, out h);
+
+                if (!isFullScreen)
+                {
+                    SDL.SDL_GetWindowPosition(Handle, out x, out y);
+
+                    x += BorderX;
+                    y += BorderY;
```

Comment:
I suspect this border calculation is unnecessary.

The docs say that `SDL_GetWindowSize()` return the size the client area... not the window size.

Also this post confirms it...

http://stackoverflow.com/a/31996739

Are you certain this is needed?